tests/unit/s2n_mem_usage_test.c

```diff
@@ -34,7 +43,15 @@
 #define MAX_CONNECTIONS 1000
 
 /* This is roughly the current memory usage per connection */
+#ifdef __FreeBSD__
+#define MEM_PER_CONNECTION (58 * 1024)
+
```

Comment:
That's somewhat alarming :\

Reply:
What that connections cost 9k more memory on FreeBSD than Linux? I was surprised by that too.